parachain/primitives/core/src/types.rs

```diff
@@ -10,17 +9,10 @@ use sp_std::vec::Vec;
 #[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo)]
 pub enum MessageId {
```

Comment:
This is non-blocking but hould we collapse this enum into a struct?

Reply:
Makes sense. It's easy enough, I'll add it here.